CWE-94-Python_darwin/askervin_python-il/il.py

```
@@ -0,0 +1,480 @@
+    if libspec == None:
+        libspec = _lib_filename()
+    if isinstance(libspec, dict):
+        lib = libspec
+    elif isinstance(libspec, str):
+        # libspec is a filename
+        if libspec in _g_loaded_libs:
+            lib = _g_loaded_libs[libspec]
+        else:
+            if os.access(libspec, os.R_OK):
+                try:
+                    lib = pickle.loads(zlib.decompress(open(libspec, "rb").read()))
```

Ruff (0.13.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)